packages/edit-widgets/package.json

```diff
@@ -48,7 +48,6 @@
 		"@wordpress/url": "file:../url",
 		"classnames": "^2.2.5",
 		"lodash": "^4.17.19",
-		"reakit": "^1.3.4",
```

Comment:
It seems that it's not being used on this package. So I just removed the dependency.

Reply:
I'm sure I missed it, thank you for catching it 👍🏻